datastore/builtin/repo_test.go

```diff
@@ -63,7 +65,8 @@ func TestRepo(t *testing.T) {
 			db.SetBuild(testRepo, &common.Build{State: "success"})
 			db.SetBuild(testRepo, &common.Build{State: "success"})
 			db.SetBuild(testRepo, &common.Build{State: "pending"})
-			db.SetLogs(testRepo, 1, 1, []byte("foo"))
+			//db.SetLogs(testRepo, 1, 1, []byte("foo"))
+			db.SetLogs(testRepo, 1, 1, io.Reader(bytes.NewBuffer([]byte("foo"))))
```

Comment:
the `bytes.Buffer` implements the `io.Reader` interface, so I'm not sure if you need to wrap it in `io.Reader`